webapp/content/js/composer_widgets.js

```diff
@@ -1088,6 +1088,7 @@ function createFunctionsMenu() {
         {text: 'Power', handler: applyFuncToEachWithInput('pow', 'Please enter a power factor')},
         {text: 'Square Root', handler: applyFuncToEach('squareRoot')},
         {text: 'Time-adjusted Derivative', handler: applyFuncToEachWithInput('perSecond', "Please enter a maximum value if this metric is a wrapping counter (or just leave this blank)", {allowBlank: true})},
+	{text: 'Delay', handler: applyFuncToEachWithInput('scale', 'Please enter the number of steps to delay')},
```

Comment:
Note this should be `delay` instead of `scale` here. I'll fix this in my branch.